serfclient/environmentconfig.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+
+class EnvironmentConfig(object):
```

Comment:
Can you add some documentation for this class please?

It doesn't have any parameters so understanding what it does is slightly more important. Here's an example:

```python
"""
Reads environment variables that Serf understands and parses
them into a easily consumable object.

Currently supports reading the following environment variables:

 - SERF_RPC_AUTH
 - SERF_RPC_ADDR
"""
```